README.md

```diff
@@ -51,6 +50,12 @@ yarn add @rocketseat/unform
 
 ## Table of contents
 
+- [Overview](#overview)
+- [Key features](#key-features)
+- [Why not Formik, Redux Form or another library?](#why-not-formik-redux-form-or-another-library)
+- [Roadmap](#roadmap)
+- [Installation](#installation)
+- [Table of contents](#table-of-contents)
```

Comment:
Also, even if we decide to keep all the links the way it is, it is a bit weird to link the table of contents inside the table of contents 😄 .

Reply:
Agree.

Reply:
ToC link removed.